pyathena/common.py

```diff
@@ -142,24 +142,53 @@ def _build_start_query_execution_request(self, query, work_group=None, s3_stagin
             })
         return request
 
-    def _execute(self, operation, parameters=None, work_group=None, s3_staging_dir=None):
+    def _find_previous_query_id(self, request, work_group, cache_size):
```

Comment:
The signature of this method is as follows:
```def _find_previous_query_id (self, query, s3_staging_dir, work_group, cache_size):```
I want the `request_params` variable to be renamed to `request`.